src/realm.js

```diff
@@ -677,24 +677,42 @@ export class Realm {
   // call.
   evaluatePure<T>(
     f: () => T,
-    reportSideEffectFunc?: (
-      sideEffectType: SideEffectType,
-      binding: void | Binding | PropertyBinding,
-      value: void | Value
-    ) => void
+    mergeWithParentEvaluatePures: boolean = false,
```

Comment:
Why does this need a default value if we're passing it explicitly?

Reply:
It shows up in Atom and VSCode, so you can decide what to use if you’re not sure. We should do this everywhere tbh